onnx/onnx-operators-ml.proto3

```diff
@@ -169,7 +168,3 @@ message OperatorSetProto {
   // The (name, version) MUST be unique across all OperatorProtos/FunctionProtos in operator/functions
   repeated FunctionProto functions = 9;
 }
-
-// For using protobuf-lite
-option optimize_for = LITE_RUNTIME;
```

Comment:
this is not what you want, I believe, please run gen_proto.py --ml --lite.

Reply:
Done. Thank you!